package/CHANGELOG

```diff
@@ -99,7 +99,7 @@ Changes
 Deprecations
 
 
-08/15/23 IAlibay, jaclark5, MohitKumar020291, orionarcher, xhgchen,
+15/08/23 IAlibay, jaclark5, MohitKumar020291, orionarcher, xhgchen,
```

Comment:
That's not a legal MM/DD/YY — I think this needs to be reversed to 08/15/23.

(Sorry that we used US convention... should have used ISO YYYY-MM-DD)

Reply:
Ahhh I think I messed up and changed the date in the wrong place sorry! Should have been line 79 🤦🏽‍♂️ teaches me to not pay attention.

Reply:
I'm confused, I based my review on the date above this one, which is `28/08/23`. Are there problems in multiple locations then?

Reply:
@tylerjereddy it's the one above that is wrong (that's what I was meaning to fix 😅 and changed the wrong one because I was in a rush) - so yes now there are two wrong dates... (sorry)

Reply:
Ah, I see, I guess Irfan did the same thing maybe, most are the other format.